app/mailers/communicart_mailer.rb

```diff
@@ -17,12 +17,22 @@ def cart_observer_email(to_email, cart)
     send_cart_email(sender, to_email, cart)
   end
 
+  def sent_confirmation_email(cart)
```

Comment:
Open to better naming suggestions, perhaps something that indicates it is going to the requester?